packages/kmod-6.1-nvidia-r580/nvidia-tesla-build-config.toml.in

```diff
@@ -0,0 +1,18 @@
+[nvidia-tesla]
+lib-modules-path = "kernel/drivers/extra/video/nvidia/tesla"
+objects-source = "__NVIDIA_MODULES__"
+
+[nvidia-tesla.object-files."nvidia.o"]
+link-objects = ["nv-interface.o", "nv-kernel.o"]
```

Comment:
I'm surprised you don't need the common module for this? https://github.com/bottlerocket-os/bottlerocket-kernel-kit/blob/develop/packages/kmod-6.12-nvidia-r580/nvidia-tesla-build-config.toml.in#L9

Reply:
Ok, just wanted to confirm it was 6.12 and not 580 related.